posthog/management/commands/test/test_migrate_kafka_data.py

```diff
@@ -1,347 +0,0 @@
-from unittest import mock
```

Comment:
can we pull this out of this PR - maybe a separate pr on master if the tests still fail

Reply:
The problem is tests do not fail. They run forever until GitHub actions times out. And it only happens when mixed in with these changes.

Reply:
So, if we separate it then:
1. In this PR tests will run forever.
2. In the other PR everything will run fine.

Reply:
Haven't had any luck in catching the bug :sweat: